src/nrfx/drivers/nrfx_common.h

```diff
@@ -209,10 +219,37 @@ NRF_STATIC_INLINE IRQn_Type nrfx_get_irq_number(void const * p_reg)
     return (IRQn_Type)NRFX_IRQ_NUMBER_GET(p_reg);
 }
 
+/**
+ * @brief IRQ handler type.
+ */
+typedef void (*nrfx_irq_handler_t)(void);
```

Comment:
Silly thing: Please keep the file as close as possible to the original => this one after NRFX_WAIT_FOR :)

Reply:
Done.